test/contracts/HCElection.aes

```diff
@@ -58,10 +67,17 @@ main contract HCElection =
     assert_protocol_call()
     put(state{ leader = leader })
 
-  stateful entrypoint step_eoe(leader : address, seed : bytes(), epoch_adjust : int) =
+  stateful entrypoint step_eoe(leader : address, seed : bytes(), epoch_adjust : int,
+                               next_base_pin_reward : int, carry_over_flag : bool) =
```

Comment:
Probably we should use `None/Some(x)` for both adjusting epoch length and base pin reward. But feel free to ignore that in this PR.

Reply:
Will do that. If we feel that this is too egregious later, I'll just redo.